utils/node_local_test.py

```diff
@@ -490,6 +492,16 @@ def __init__(self, conf, test_class=None, valgrind=False):
         else:
             self.dfuse_cores = None
 
+    def __enter__(self):
+        self.start()
+        return self
+
+    def __exit__(self, type, value, traceback):
```

Comment:
(pylint-redefined-builtin) Redefining built-in 'type'
(pylint-redefined-outer-name) Redefining name 'traceback' from outer scope (line 30)